src/main.rs

```diff
@@ -7,25 +7,28 @@ use diss::{list_sessions, run};
 #[clap(author, version, about, long_about = None)]
 struct Args {
     /// list sessions
-    #[clap(short, long, value_parser)]
+    #[clap(short, long)]
     list: bool,
 
     // debug
-    #[clap(short, long, value_parser)]
+    #[clap(short, long)]
     debug: bool,
 
     // escape key
-    #[clap(short, long, value_parser)]
+    #[clap(short, long)]
     escape_key: Option<String>,
 
     // session name
-    #[clap(short, long, value_parser)]
+    #[clap(short, long)]
+    #[clap(required = true)]
     attach_session: Option<String>,
```

Comment:
I am not sure if `--attach-session` makes so much sense, I'd call it `--session-name`. @yazgoo what do you think?

Reply:
yes, why not

Reply:
I was thinking of separate PR, but perhaps it doesn't matter so much.
Do you care about a gradual transition with `--attach-session` being marked as deprecated etc?